### PR TITLE
Add Windows Master config information

### PR DESCRIPTION
**Pull request by twangboy**
### What does this PR do?
Adds file location translation for Windows master file location configuration options

### What issues does this PR fix or reference?
https://github.com/saltstack/zh/issues/1316

### Tests written?
NA

### CODE REVIEW

**Review by vutny — CHANGES_REQUESTED** [review]
Salt Master on Windows... wow.

I just have some style comment about nodegroups section.

File: doc/ref/configuration/master.rst [hunk 1]
```diff
@@ -3972,13 +3988,15 @@ A group consists of a group name and a compound target.
       group2: 'G@os:Debian and foo.domain.com'
       group3: 'G@os:Debian and N@group1'
       group4:
-	- 'G@foo:bar'
-	- 'or'
-	- 'G@foo:baz'
+    - 'G@foo:bar'
+    - 'or'
+    - 'G@foo:baz'
```
Comment (cachedout) [inline doc/ref/configuration/master.rst]: I believe @vutny is correct. This needs to be changed.

**Review by cachedout — CHANGES_REQUESTED** [review]

File: doc/ref/configuration/master.rst (hunk 1 above)
Comment (twangboy) [inline doc/ref/configuration/master.rst]: I believe PyCharm did an automatic conversion from tabs to spaces and it jacked this one up... I'll get it fixed.

File: doc/ref/configuration/master.rst [hunk 2]
```diff
@@ -3994,9 +4012,11 @@ https://github.com/ytoolshed/range/wiki/%22yamlfile%22-module-file-spec
 
 .. code-block:: yaml
 
-  range_server: range:80
+    range_server: range:80
```
Comment (twangboy) [inline doc/ref/configuration/master.rst]: This was a tab... caught by PyCharm. The same thing that fixed this, broke the one above...

**Review by cachedout — APPROVED** [review]

**Pull request merged**